lib/install/config/shared.js

```diff
@@ -53,3 +59,10 @@ module.exports = {
     modules: [ path.resolve('../vendor/node_modules') ]
   }
 }
+
+module.exports = () => { 
+  return {
+    distPath: distPath,
+    config: config
+  }
+}
```

Comment:
I don't see any reason to return a function here.  It's not like these config values will/should change.